README.md

```diff
@@ -1,6 +1,11 @@
 [![][ButlerImage]][website] 
 
 # About
+
+[![Weekly Release](https://img.shields.io/badge/dynamic/yaml.svg?url=https://raw.githubusercontent.com/jenkins-infra/jenkins.io/master/content/_data/changelogs/weekly.yml&label=Weekly%20Release&query=$[-1:].version&color=green)](https://jenkins.io/changelog/)
```

Comment:
This is amazing.

Reply:
Have you tried probing this endpoint for SSRF exploits? :D

Reply:
feel free to do so :P